src/dotnet/APIView/APIView/HTMLRendererAPIV.cs

```diff
@@ -40,6 +40,18 @@ protected override void RenderConstructor(StringBuilder builder, MethodAPIV m)
                 .Append(EscapeHTML(m.Name)).Append("</a>");
         }
 
+        protected override void RenderEvent(StringBuilder builder, EventAPIV e)
+        {
+            builder.Append("<a id=\"").Append(e.Id).Append("\" class=\"name commentable\">").
+                Append(e.Name).Append("</a>");
+        }
+
+        protected override void RenderField(StringBuilder builder, FieldAPIV f)
+        {
+            builder.Append("<a id=\"").Append(f.Id).Append("\" class=\"name commentable\">").
```

Comment:
Two things:
1. Naming: Id vs NavigationID
2. All these methods do very similar or event the same thing, can we unify them?